### DISCUSSION TITLE
error by improve_prompt

### DISCUSSION THREAD

**Discussion by lionardben211** [danielmiessler/Fabric#574] [Q&A]
I am getting error by this pattern.
![2024-06-08 23_22_37-Command Prompt](https://github.com/danielmiessler/fabric/assets/36739989/dce553fc-3edc-42f3-8cc0-7dd5d0302ef0)

**Comment by kayozxo** [comment]
You need WSL (Windows Subsystem for Linux) for fabric to work.
